### DISCUSSION TITLE
Issues to get started development

### DISCUSSION THREAD

**Discussion by mbedded** [zidoro/pomatez#404] [Q&A] [answered]
Hello everybody :)

I wanted to take some time and (maybe) fix the issue by myself which I reported. I read the guide to get the application up and running but I miss something or the documentation (Readme) could be enhanced:

The quick setup says
```
yarn install or npm install
yarn dev:app or npm run dev:app
```

I interpret the **or** as "I can use yarn or npm". I haven't installed yarn on my local machine because I don't do that much with NodeJS so npm is sufficient for my cases. So I am just running `npm install` and get an exception:

```
npm ERR! code 127
npm ERR! path /mnt/volume1/documents/projects/pomatez/app/main
npm ERR! command failed
npm ERR! command sh -c yarn build
npm ERR! sh: 1: yarn: not found
```

I used the following branches: `master` (74f2f09) and `revamp` (e823e2e) and both returned errors.
master -> yarn issue
revamp -> cannot resolve some dependencies

So my questions are basically the following:
- Do I miss something? Do I have to install yarn? (Then it should be added somewhere as _prerequisite_ in the Readme file)
- Is the package definition invalid? Can yarn be added as a package to avoid this error for other users which are only using npm?

I have no experience with yarn so far so I am happy if you can give me some hints to get it up and running :)

Best regards
Matthias

**Comment by roldanjr** [comment, marked as answer]
Hi @mbedded sorry about that. My bad, this project uses `yarn` and it's a prerequisite.
I'll update the readme to fix the misleading instructions.

In the meantime, install `yarn` globally and use it to install the dependencies.

**Reply by roldanjr** [reply]
Here's the updated [Quick Setup](https://github.com/zidoro/pomatez#zap-quick-setup) section for your reference @mbedded

**Reply by mbedded** [reply]
Hi @roldanjr, thank you for your quick response and feedback :)
I assumed that I have to install yarn because of the error message, I just wanted to ensure that I don't miss anything so that I am able to go the right way.

Thank you for updating the readme so quickly. I think it helps many developers to get started and support the development of this application. Now I can start learning react and (hopefully) fixing my bug :)
